build_daemon/lib/client.dart

```diff
@@ -60,6 +60,9 @@ Future<void> _handleDaemonStartup(
 bool _isActionMessage(String line) =>
     line == versionSkew || line == readyToConnectLog || line == optionsSkew;
 
+/// A client of the build daemon.
+///
+/// Handles starting and connecting to the build daemon.
```

Comment:
We should add some more docs on general usage here, maybe a full on basic example of registering a target and building it. This could highlight the differences between doing a single build and multiple builds.